setup.py

```diff
@@ -72,7 +72,8 @@
             "Flask>=1,<2",
             "Werkzeug<2",  # TODO: support Flask 2.x
             "pyramid>=1,<2",
-            "sanic>=20,<21",
+            "sanic>=21,<22",
+            "sanic-testing>=0.6",
```

Comment:
Since sanic v21, the framework separated its testing support to a different PyPI package.